test/sms_nurse.test.js

```diff
@@ -13,23 +18,114 @@ describe("app", function() {
             tester = new AppTester(app);
 
             tester
+                .setup.char_limit(160)
                 .setup.config.app({
-                    name: "sms_nurse",
-                    env: "test"
+                    name: "nurse_sms",
+                    testing: "true",
+                    testing_today: "April 4, 2014 07:07:07",
+                    testing_message_id: "0170b7bb-978e-4b8a-35d2-662af5b6daee",
+                    channel: "longcode",
+                    nurse_ussd_channel: "nurse_ussd_channel",
+                    services: {
+                        identity_store: {
+                            url: "http://is/api/v1/",
+                            token: "test IdentityStore"
+                        },
+                        stage_based_messaging: {
+                            url: "http://sbm/api/v1/",
+                            token: "test StageBasedMessaging"
+                        },
+                    },
+                    logging: "off"
                 })
+                .setup(function(api) {
+                    // add fixtures for services used
+                    // fixtures_Hub().forEach(api.http.fixtures.add); // fixtures 0 - 49
```

Comment:
These should not be commented out else the numbering is all off and merge conflicts reign